platform/jobs/edxPlatformUnitCoverage.groovy

```diff
@@ -176,7 +179,10 @@ secretMap.each { jobConfigs ->
                     }
                 }
             }
-            shell('./scripts/jenkins-report.sh ' + jobConfig['shellKey'] + ' $CI_BRANCH')
+            // Run jenkins-report.sh which will upload coverage results to
+            // both codecov and coveralls. This is temporary. We plan to move
+            // off of coveralls.
```

Comment:
@estute I think I'm missing something here. With a refactor of jenkins-report.sh (over in the edx-platform repo), now that we are passing the CODE_COV token shouldn't we be able to control from that repo now whether the coverage results get uploaded to codecov, coveralls, both, or neither?